src/ducks/Studio/Template/Button.js

```diff
@@ -1,10 +1,57 @@
 import React, { useState } from 'react'
 import cx from 'classnames'
 import Icon from '@santiment-network/ui/Icon'
+import Tooltip from '@santiment-network/ui/Tooltip'
 import FormDialogNewTemplate from './Dialog/NewTemplate'
 import LoginDialog from '../../../components/LoginDialog'
 import styles from './index.module.scss'
 
+const TooltipWrapper = ({ selectedTemplate, children }) => {
+  if (!selectedTemplate) {
+    return children
+  }
+
+  return (
+    <Tooltip
+      trigger={children}
+      position='bottom'
+      align={'start'}
+      className={styles.tooltip}
+    >
+      Click to save '{selectedTemplate.title}'
+      <svg
+        fill='none'
+        xmlns='http://www.w3.org/2000/svg'
+        className={styles.tooltipIcon}
+      >
+        <path d='M24.69 25.44a2 2 0 01-2.93 0L1.24 3.38A2 2 0 012.7.02h41.05a2 2 0 011.46 3.36L24.7 25.44z' />
+      </svg>
```

Comment:
If it's arrow for tooltip - let's try to reuse it, because as I understand correctly - we need it in a lot of places now..

Reply:
in future will be reusable ;)